template/module/static/src/js/module_name.js

```diff
@@ -1,7 +1,20 @@
 /* Copyright <YEAR(S)> <AUTHOR(S)>
  * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
 
-(function ($) {
+odoo.define('module_name.object_name', function(require){
     "use strict";
-    // Script that will be loaded when document is ready
-})(jQuery);
+
+    var core = require('web.core');
+
+    core.bus.on('web_client_ready', null, function () {
+        // Script that will be loaded when document is ready
+    });
```

Comment:
This is not used in website, but on web client (as stated). Maybe we should include both in 2 files: one for web, and another for website.